.circleci/config.yml

```diff
@@ -427,13 +548,28 @@ workflows:
             tags:
               only: /.*/
 
-      - build-packages:
+      - build-packages-opencv-python:
           requires:
             - build-packages-no-numpy-dependents
           filters:
             tags:
               only: /.*/
 
+      - build-packages-numpy-dependents:
+          requires:
+            - build-packages-no-numpy-dependents
+          filters:
+            tags:
+              only: /.*/
+
+      - build-packages:
+          requires:
+            - build-packages-numpy-dependents
+            - build-packages-opencv-python
```

Comment:
It might be good to eventually split the package tests up too, so that we can do `no-numpy-dependents` tests before scipy and opencv finish building. This would make working on simple packages significantly more ergonomic -- it takes a long time for package tests to come back currently.

Reply:
Yeah, that would be really great. We can run `no-numpy-dependents` test before building large numpy dependent packages, but we will need a minor fix to not run the same tests twice.

Reply:
>  need a minor fix to not run the same tests twice

Yeah that's why I haven't done this...